specs/agents/tracing-instrumentation-aws.md

```diff
@@ -43,17 +43,27 @@ The format should be `DynamoDB <ActionName> <TableName>`. So for example, `Dynam
 
 ### SQS (Simple Queue Service)
 
-AWS Simple Queue Service is a message queuing service. The [messaging spec](tracing-instrumentation-messaging.md) can 
+AWS Simple Queue Service is a message queuing service. The [messaging spec](tracing-instrumentation-messaging.md) can
 be used for instrumenting SQS, but the follow specifications supersede those of the messaging spec.
 
 - **`context.destination.cloud.region`**: mandatory. The AWS region where the queue is.
 
+#### Distributed Tracing
+
 For distributed tracing, the SQS API has "message attributes" that can be used in lieu of headers.
 
+Agents should use an attribute name of `Traceparent` or `Elastic-Trace-Parent` when sending the traceparent headers via the SQS message attributes.  Agents should use an attribute name of `Tracecontext` or `Elastic-Trace-Context` if sending tracecontext headers in an SQS message attribute.
```

Comment:
Is there a particular reason `Traceparent` is capitalized? According to the w3c spec, it SHOULD be lower case to enhance interoperability (see https://www.w3.org/TR/trace-context-1/#header-name).

Reply:
I think it comes from setting it as HTTP headers, commonly Camel-Cased or whatever it's called when capitalized and dash-separated.

Makes less sense to do here.

Reply:
> I think it comes from setting it as HTTP headers, commonly Camel-Cased

While this is commonly the case, the w3c spec specifically states that the header name SHOULD be lower-case. It's not a big deal as headers are usually case-insensitive. But I think we should follow the recommendation and set lower-cased values.

Reply:
Latest version with lower cased values: https://github.com/elastic/apm/blob/fb5686893f2c6683d7e79950f72c4764dfc4052c/specs/agents/tracing-instrumentation-aws.md#distributed-tracing